doc/developer-guide/developer-guide.rst

```diff
@@ -32,6 +33,39 @@ rotgui
 * **MainWindow** is the main window of the application.
 * **Model** contains the main business logic in the application.
 * **ControlPanel** has the commands to control the controller.
+* **Config** is a data class that has the configuration details in the rotator control system.
+* **Status** is a data class that has the current controller status.
+
+The model–view–controller (MVC) architecture is used in this module.
+In this design, the view always shows the data sent from the model.
+This helps to minimize the business logic in view and makes the tests easier.
```

Comment:
```suggestion
This helps minimize the business logic in view and makes testing easier.
```

Reply:
Improved in the commit: [9135b21](https://github.com/lsst-ts/ts_rotgui/pull/3/commits/9135b21900f0f24ecdb4dc0b5f0b99fd9a2a9b39). Thanks!